modules/configuring-delegated-image-scanning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="configuring-delegated-image-scanning_{context}"]
+= Configuring delegated image scanning
+
+A new delegated registry configuration specifies the registries from which image scans are to be delegated. For images observed by Sensor, this configuration allows you to delegate scans from no registries (default), all registries, or specific registries. To enable delegation of scans using the `roxctl` CLI, the Jenkins plugin, or the API, you must also specify a destination cluster and source registry.
+
+
+.Prerequisites
+
+* Scanner-slim must be installed in the secured cluster to scan images.
++
+[NOTE]
+====
+Enabling Scanner-slim is supported on {ocp} and Kubernetes secured clusters.
+====
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters*.
+. In the *Clusters* view header, click *Manage delegated scanning*.
+. In the *Delegated Image Scanning* page, provide the necessary information:
```

Comment:
Suggestion: replace "necessary" with "following", because if something in a procedure is not labeled as optional then it is implied to be necessary, and also because one of the nested list items is optional.